src/components/elements/MapPolygonPanel/VersionInformation.tsx

```diff
@@ -0,0 +1,116 @@
+import classNames from "classnames";
+import { Dispatch, SetStateAction } from "react";
+
+import Icon, { IconNames } from "@/components/extensive/Icon/Icon";
+import ModalConfirm from "@/components/extensive/Modal/ModalConfirm";
+import { useModalContext } from "@/context/modal.provider";
+
+import Menu from "../Menu/Menu";
+import { MENU_PLACEMENT_RIGHT_BOTTOM } from "../Menu/MenuVariant";
+import Text from "../Text/Text";
+
+const VersionInformation = ({ setPreviewVersion }: { setPreviewVersion: Dispatch<SetStateAction<boolean>> }) => {
+  const { openModal, closeModal } = useModalContext();
+  const openFormModalHandlerConfirm = () => {
+    openModal(
+      <ModalConfirm
+        title={"Confirm Version Delete?"}
+        content="Do you want to delete this version?"
```

Comment:
`t()` in this component

Reply:
done

Reply:
These two still look like they need `t()`, but I see it was added in a lot of other places in the component.

Reply:
done